src/websocket.c

```diff
@@ -326,6 +331,7 @@ websocket(void *arg)
 	  write_events = events;
 	  events = 0;
 	  lws_callback_on_writable_all_protocol(context, &protocols[WS_PROTOCOL_NOTIFY]);
+	  events_counter++;
```

Comment:
Increment the `write_events_counter` after the assignment of `write_events = events`. That way we start with 1 for the first event notification which makes it a bit easier in the callback (no decrement necessary).

Reply:
Thanks for the review..made the requested changes